docs/encoding/svtav1.md

```diff
@@ -0,0 +1,465 @@
+therefore it is usually not recommended to stray away from the defaults.
+
+### Constrained Directional Enhancement Filter
+
+According to the [SVT-AV1 documentation](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Appendix-CDEF.md),
+*"The constrained directional enhancement filter (CDEF) [...] aims at improving the reconstructed picture by addressing ringing artifacts."*
+
+The closest equivalent to this feature would be SAO in x265,
+except it is not prone to the same level of detail loss.
+
+`--enable-cdef` tends to limit edge artefacts effectively
+without introducing severe *additional* line fading,
+thus improving visual appeal.
+
+Therefore, it is **recommended to leave it on.**
+
+??? info "Specific SVT-AV1-HDR usage"
+
+     The *SVT-AV1-HDR* fork disables CDEF in *tune 3* to hopefully increase
+     grain retention consistency across the whole picture, but it is effectively
+     a bruteforcing method and cannot be generalized to a more general usecase.
+
+### Restoration Filter
+
+According to the [SVT-AV1 documentation](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Appendix-Restoration-Filter.md),
+*"The restoration filter [...] aims at improving the reconstructed picture by recovering some of the information lost during the compression process."*
+
+In effect, `--enable-restoration` tends to increase efficiency
+slightly and doesn't have any documented drawbacks, 
+so it **can safely be left on** at all times.
+
+??? info "Specific SVT-AV1-HDR usage"
+
+     The *SVT-AV1-HDR* fork disables restoration in *tune 3* to hopefully increase
+     grain retention consistency across the whole picture, but it is effectively
+     a bruteforcing method and cannot be generalized to a more general usecase.
+
+### Temporal Filtering
+
+Temporal filtering in SVT-AV1 combines information from multiple nearby
+video frames to create cleaner reference pictures with reduced noise, 
+which helps improve compression quality especially for noisy source material.
+
+The feature was often considered too strong and often created unavoidable
+blocking on keyframes, so we historically disabled temporal filtering.
+However *SVT-AV1-PSY* introduced a strength parameter
+which has allowed to tame its effects. `--tf-strength` was backported
+into mainline SVT-AV1, so it is widely available.
+
+It is recommended to **leave `--enable-tf` on** for the efficiency benefits
+it provides, but to **reduce `--tf-strength`** from its default `3` **to `1`**,
+or below, to completely eliminate the tf blocking issue.
+
+??? info "Specific SVT-AV1-PSY forks usage"
+
+     The *SVT-AV1-PSY(EX)* and *-HDR* forks include an additional
+     `--kf-tf-strength` which decouples tf strength on
+     keyframes, and allows the user to concurrently 
+     fix the blocking issue and use a stronger 
+     tf strength on all other frames
+     if one wishes so. 
+
+For the rest of this section from this point on, it is less clear

```

Comment:
This may benefit from having a warning admonition, or some other way to clearly separate it from the previous (such as an additional header or \<hr>)?